pulser-core/pulser/sequence/helpers/_switch_device.py

```diff
@@ -16,6 +16,7 @@
 
 import itertools
 import warnings
+from dataclasses import asdict
```

Comment:
nit: `asdict` is a bit too vague, I would rather have an `import dataclasses` and then a call to `dataclasses.asdict()`